etc/profile-m-z/newsboat.profile

```diff
@@ -8,6 +8,10 @@ include globals.local
 
 noblacklist ${HOME}/.newsboat
 
+# Lynx Support
+private-bin gzip,lynx,sh
+private-etc gzip,lynx,lynx.cfg,lynx.lss,sh
```

Comment:
Like I [asked](https://github.com/netblue30/firejail/issues/3894#issuecomment-760575889) on the issue report, are you sure private-etc needs gzip, lynx and sh? Do these paths exist on your Arch installation?